panther_lights_scheduler/src/scheduler_node.py

```diff
@@ -0,0 +1,120 @@
+#!/usr/bin/env python3
+
+import rospy
+
+from sensor_msgs.msg import BatteryState
+from std_msgs.msg import Bool
+
+from panther_msgs.msg import LEDAnimation
+from panther_msgs.srv import SetLEDAnimation, SetLEDAnimationRequest
+
+
+class LightsSchedulerNode:
+    def __init__(self, name) -> None:
+        rospy.init_node(name, anonymous=False)
+
+        self._battery_percentage = 1.0
+        self._charger_connected = False
+        self._e_stop_state = None
+        self._led_e_stop_state = None
+        self._last_battery_anim_time = 0.0
```

Comment:
```suggestion
        self._last_battery_anim_time = rospy.get_time()
```
I would initialize it this way

Reply:
Thought about it but if battery state will be low/critical on start, then node will wait for `*_anim_period` to change animation which I don't think is optimal.

Reply:
I'm not sure if usage of timers won't solve this problem